CHANGELOG.md

```diff
@@ -1,10 +1,25 @@
 # Changelog
 
+## 0.41.2      2021-11-18
+
+* Add two additional log_line_prefix settings
+  - `%p-%s-%c-%l-%h-%u-%d-%m `
+  - `%m [%p][%b][%v][%x] %q[user=%u,db=%d,app=%a] `
+* Change schema collection message to warning during test run
+  - This helps discover schema collection issues, e.g. due
+    to connection restrictions or pother ermission problems
+* Add support for multiple domain constraints
```

Comment:
This is technically "fix issue with" since multiple constraints would break schema collection entirely before, but NBD.

Reply:
Good point, adjusted